src/ctapipe/vectorization/aggregate.py

```diff
@@ -0,0 +1,145 @@
+"""Helper functions for vectorizing numpy operations."""
+
+import numpy as np
```

Comment:
Do we really need a full new submodule for this? This looks like it could be in `ctapipe/reco/telescope_event_handling` or similar

Reply:
I think I put it in `ctapipe.utils` initially, but that module is already pretty large. 
But that would be fine with me. Alternatively, what about `ctapipe.reco.utils`?

Reply:
Sure, that also fits

Reply:
Maybe just make a better-named, potentially private, sub-module under `reco` or `io`. Dumping random things into modules called "Utils" ends up being too generic (see #1110).

Reply:
Good point. I went with Max' initial idea of `ctapipe.reco.telescope_event_handling`. Regarding the docs: Currently only the different reconstructors themselves are included in the docs, but none of the different helper/ preprocessing functions. Do we want to keep it this way?